### PR TITLE
Define bugs

### PR DESCRIPTION
**Pull request by victorwuky**
I (Victor) don't think these are bugs:
- https://github.com/windmilleng/tilt/issues/1852
- https://github.com/windmilleng/tilt/issues/1910
- https://github.com/windmilleng/tilt/issues/2081

I think they result in poor user experiences nonetheless. I think it's important to define what a bug is, and distinguish it from poor UX (which is what I would categorize the above).

I think the following are bugs:
- https://app.clubhouse.io/windmill/story/4266
- https://app.clubhouse.io/windmill/story/4265
- https://app.clubhouse.io/windmill/story/3760

### CODE REVIEW

**Comment by maiamcc** [comment]
Yes, I agree with the distinction, and think it's important. I think those got labeled as "bugs" because the only two labels we use with any frequency are "bug" and "enhancement" (the latter generally referring to NEW FEATURES) and we don't really have a third grouping for "not a bug but not a new feature but something that should probably be fixed...". I think if we had a good label for that middle category, that'd keep us from mis-categorizing bugs. What would you call that?

(Also, were there supposed to be content changes in this PR?)

**Comment by victorwuky** [comment]
> (Also, were there supposed to be content changes in this PR?)

@maiamcc : I purposely created an empty PR here for a more free-flowing discussion, since I don't have a specific change proposal just yet, but I expect over these convos we can hopefully converge on something specific. So instead of talking in a separate space (issue/story), everything will be self-contained in place!

**Comment by victorwuky** [comment]
@maiamcc @nicks : Does the updated PR diff make sense to you? I think this should be pretty uncontroversial as an incremental first step. And then we could use https://github.com/windmilleng/company/pull/4 to further add sev and/or SLA definitions to iterate on the stated process (from the working agreement).

I didn't bother adding any other categories like "feature request", "tech debt", "enhancement", etc. etc. I don't necessarily think it's important to burden ourselves with over-specifying things since we are still a small team. But I think having a crisp definition of bug is important for prioritization so that we align ourselves during collaborative decision-making.

File: development/README.md [hunk 1]
```diff
@@ -1 +1,9 @@
-_Development processes to be added, like code review_
+# Development
+
+## Bug
+
+We define a bug as any experience that breaks the _intended_ functionality of a user experience in [Tilt](https://docs.tilt.dev/), including [Tilt Cloud](https://cloud.tilt.dev/). In particular, we may have shipped a feature that failed to meet a certain user's expectation, but that may not necessarily qualify as a bug. In this case, we may should nonetheless track the user problem as a non-bug to improve Tilt.
+
+We should express our intentions through [documentating Tilt functionality](https://docs.tilt.dev/) exhaustively, to better support users and manage their expectations.
+
+Non-user facing problems are not classified as bugs.
```
Comment (maiamcc) [inline development/README.md]: Can you give an example? Not sure I follow.

**Comment by maiamcc** [comment]
> I didn't bother adding any other categories like "feature request", "tech debt", "enhancement", etc. etc. I don't necessarily think it's important to burden ourselves with over-specifying things since we are still a small team.

This makes sense, but I think we might need a procedure change elsewhere, because part of the Exterminator's contract (iirc) is to sort every incoming issue into either "bug" or "enhancement" (i.e. feature request). We should either add another category(s), or remove this from the Exterminator role description.

Nice doc!

**Comment by victorwuky** [comment]
Yes that makes sense thanks for the reminder @maiamcc

**Pull request merged**